ThermoRawFileReader/ThermoDynamicData.cs

```diff
@@ -76,7 +76,6 @@ private void InitiateDynamicConnection(string filePath)
             }
 
             dynamicConnection.SelectInstrument(Device.MS, 1);
-            GetMsOrdersByScanInDynamicConnection();
```

Comment:
?

Reply:
GetMsOrdersByScanInDynamicConnection is already called on line 20 when creating a dynamic connection. i just deleted this line because it's a duplicate call and it's useless anyway (the returned data is not saved)